WooCommerce/UITestsFoundation/XCTest+Extensions.swift

```diff
@@ -114,8 +114,44 @@ extension XCUIElement {
         startCoordinate.press(forDuration: 0.01, thenDragTo: destination)
     }
 
-    public func getTextVisibilityCount(text: String) throws -> Int {
-        let predicate = NSPredicate(format: "label CONTAINS[c] %@", text)
+    func getStaticTextVisibilityCount(textToFind: String) throws -> Int {
```

Comment:
Also not sure if this file is the right "home" for these, I'm open to feedback on this.

Reply:
These look like useful utility methods that we might want to move to [XCUITestHelpers](https://github.com/Automattic/XCUITestHelpers/).

For the time being, I think they are okay in this file which contains a loose collection of test helpers.

If we wanted to be picky, we might move all those that access `XCUIApplication` directly in an `XCUIApplication+Helpers.swift` file, or something with a similar name. But, I'd rather spend the time moving them to the helper framework.